editor/post-title/index.js

```diff
@@ -119,6 +119,7 @@ export default connect(
 		return {
 			onUpdate( title ) {
 				dispatch( editPost( { title } ) );
+				dispatch( editPostTitle( title ) );
```

Comment:
Could we just rely on the `editPost` action instead and this beeing a side-effect if the `title` is in the `edits` maybe?